### PR TITLE
Refactor: Improve Swagger Docs description

### LINKED ISSUE #482: Improve the description of the app on the Swagger UI docs
## Description

As a contributor,
I need some more description on Swagger docs,
so that I can easily understand what the app is about.

Maybe you could add the link to the wiki, and the full description of the app, link to Android app.
These are just ideas.. I'm not sure what is the best, but I know this can definitely contain some more information.

## Mocks

I tried this out myself, staying something like:
image

### Update [Required]
- [ ] Improve Swagger Docs description

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
1 hours

### PR DESCRIPTION
**Pull request by EfthymiaKostaki**
### Description
Updated Swagger Docs description for including more information about the project. Also deletes default Namespace which is not needed in the project.

Fixes #482

UI

### Type of Change:
- User Interface

### How Has This Been Tested?
When the server is running the updates are immediately visible.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

### CODE REVIEW

**Comment by isabelcosta** [comment]
@EfthymiaKostaki happy to see you figured out how to solve this issue :) Can you please include a screenshot of how the Swagger UI description looks like now? As per our contribution guidelines, when you do a change that involves UI change, its required that you include an image on the PR description. Can you please do that?

**Comment by EfthymiaKostaki** [comment]
@isabelcosta  Thank you for your comment. I edited my pull request and added a screenshot of the updated description. Is there anything else I could improve?

**Comment by isabelcosta** [comment]
I will wait until you do the removal of the Default namespace, and then I can properly review @EfthymiaKostaki

**Comment by EfthymiaKostaki** [comment]
@isabelcosta  Hello, I have updated this pull request with the commit for the default namespace issue.

**Review by naveennvrgup — DISMISSED** [review]
Checked the PR locally. Fixes the issue as descripted.

File: app/api/api_extension.py
Comment (isabelcosta) [inline app/api/api_extension.py:9]: Could you add before this line the description of the app?

**Here it is:** Mentorship System is an application that matches women in tech to mentor each other, on career development, through 1:1 relations during a certain period of time.

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@EfthymiaKostaki this is great, I just suggest adding the little description for the app there for a little more context :)

**Comment by isabelcosta** [comment]
@naveennvrgup thank you a lot for reviewing and testing this! I would love to see more of this from contributors :) I really appreciate your testing here. I think I will start adding a label to issues that need testing to guide you all.

**Comment by EfthymiaKostaki** [comment]
@naveennvrgup Thank you for your review! I don't know why GitHub shows message I dismissed your technical review, maybe it's because I added a new commit message.

@isabelcosta I have added the description of the app.

**Comment by isabelcosta** [comment]
> @naveennvrgup Thank you for your review! I don't know why GitHub shows message I dismissed your technical review, maybe it's because I added a new commit message.

It is because of the new commit, so that is ok ;)

> @isabelcosta I have added the description of the app.

Thank you so much 👏

**Review by isabelcosta — APPROVED** [review]
Great work @EfthymiaKostaki 👏 
Thank you for your contribution!

**Comment by rpattath** [comment]
Tested the changes and seeing the updated documentation.

Screenshot from 2020-04-07 16-10-41

**Comment by isabelcosta** [comment]
If you wish @rpattath you can leave a review like a comment, without approval. I know that GitHub counts that as a contribution, whereas comments do not count. I would love to these contributions of yours recorded as such.

image

**Comment by isabelcosta** [comment]
Very very good @rpattath thank you so much for your testing :D

**Comment by isabelcosta** [comment]
@foongminwong could you please review this and if you do merge, can you make sure the commit follows the guideline and includes the id of the PR?

**Comment by rpattath** [comment]
> If you wish @rpattath you can leave a review like a comment, without approval. I know that GitHub counts that as a contribution, whereas comments do not count. I would love to these contributions of yours recorded as such.
> 
> image

I did not know that, thank you for the tip !

**Comment by isabelcosta** [comment]
feel free to try that out here if you want and then check on your GitHub profile if you have an activity item listed saying you "Reviewed a pull request"

**Comment by isabelcosta** [comment]
@EfthymiaKostaki PR was merged 👏 thank you for your contribution here 🎉

**Pull request merged**
